client/src/pages/SignUp/SignUp.tsx

```diff
@@ -40,32 +39,24 @@ export default function Register(): JSX.Element {
   };
 
   return (
-    <div>
+    <>
       <LandingNavbar />
-      <Grid container component="main" className={classes.root}>
+      <Grid container component="main" className={classes.root} justify="center">
         <CssBaseline />
-        <Grid item xs={12} elevation={6} component={Paper} square>
-          <Box className={classes.authWrapper}>
-            <Box width="100%" maxWidth={450} p={3} alignSelf="center">
-              <Grid container>
-                <Grid item xs>
-                  <Typography className={classes.welcome} component="h1" variant="h5">
-                    Sign up
-                  </Typography>
-                </Grid>
+        <Grid item xs={12} sm={8} md={6} elevation={6} component={Paper} square>
+          <Box width="100%" maxWidth={350} p={3} marginY={0} marginX="auto">
```

Comment:
As mentioned with the Login form this can be consolidated to just `margin="auto"`